test/utils/helpers.test.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@esm-bundle/chai';
+import { overrideUrlOrigin } from '../../libs/utils/helpers.js';
+
+describe('overrideUrlOrigin', () => {
+  it('Change origin to http://localhost:2000', async () => {
```

Comment:
nit: I would not use a non prod origin for that test

Reply:
Checking utils is the first thing I did and I could not find it. Regarding origin, I tried to modify location.href to prod origin, didn't go very well and then I realized that localhost:2000 in many tests, so I did the same.